sources/i64.spec.move

```diff
@@ -182,21 +197,39 @@ spec move_int::i64 {
 
     spec pow {
         pragma opaque;
+        // Limits to 2 unrolls of the while loop.
+        // If a spec function is defined in a recursive way, when the while loop in the corresponding non-recursive
+        // move function is expected to execute more than certain times, SMT solver cannot prove they are equivalent.
+        pragma unroll = 2;
```

Comment:
lets add https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-prover/tests/sources/functional/math8.move#L74 as a reference in comment